infra/src/test/kotlin/fi/fta/geoviite/infra/ui/testgroup1/LocationTrackDialogTestUI.kt

```diff
@@ -0,0 +1,28 @@
+package fi.fta.geoviite.infra.ui.testgroup1
+
+import fi.fta.geoviite.infra.ui.SeleniumTest
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import kotlin.test.assertEquals
+
+@ActiveProfiles("dev", "test", "e2e")
+@SpringBootTest
+class LocationTrackDialogTestUI @Autowired constructor() : SeleniumTest() {
+
+    @Test
+    fun `New location track owner is set to vayla regardless of caching`() {
```

Comment:
Tämä tuntuu kyllä aika detaljitason asialta UI-testillä testattavaksi. Jos näin pienet tehdään e2e:nä, niitä kertyy äkkiä aika paljon. Meillä ei ikävä kyllä ole fronttipuolelle unit testejä setupattu vieläkään, mutta tämä menisi ehkä sellaisessa paremmin.